docs/tools/vdb_table/data/pgvecto_rs.json

```diff
@@ -0,0 +1,181 @@
+{
+    "name": "pgvecto.rs",
+    "links": {
+      "docs": "https://docs.pgvecto.rs/",
+      "github": "https://github.com/tensorchord/pgvecto.rs",
+      "website": "https://docs.pgvecto.rs/",
+      "vendor_discussion": "",
+      "poc_github": "https://github.com/kemingy",
+      "slug": "pgvecto.rs"
+    },
+    "oss": {
+      "support": "full",
+      "source_url": "",
+      "comment": ""
+    },
+    "license": {
+      "value": "Apache-2.0",
+      "source_url": "https://github.com/tensorchord/pgvecto.rs/blob/main/LICENSE",
+      "comment": ""
+    },
+    "dev_languages": {
+      "value": [
+        "rust"
+      ],
+      "source_url": "",
+      "comment": ""
+    },
+    "vector_launch_year": 2023,
+    "metadata_filter": {
+      "support": "full",
+      "source_url": "",
+      "comment": ""
+    },
+    "hybrid_search": {
+      "support": "full",
+      "source_url": "https://docs.pgvecto.rs/use-case/hybrid-search.html",
+      "comment": ""
+    },
+    "facets": {
+      "support": "",
+      "source_url": "",
+      "comment": ""
+    },
+    "geo_search": {
+      "support": "full",
```

Comment:
can you mark this as `partial` since the support is through a separate extension?